lib/src/sep/0099/hotswap.dart

```diff
@@ -0,0 +1,173 @@
+// Copyright 2020 The Stellar Flutter SDK Authors. All rights reserved.
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:stellar_flutter_sdk/stellar_flutter_sdk.dart';
+
+class Hotswap {
+  late String _hotswapServerAddress;
+  late http.Client httpClient;
+
+  Hotswap(
+    this._hotswapServerAddress, {
+    http.Client? httpClient,
+  }) {
+    this.httpClient = httpClient == null ? http.Client() : httpClient;
+  }
+
+  static Future<Hotswap> fromDomain(
+    String domain, {
+    http.Client? httpClient,
+    Map<String, String>? httpRequestHeaders,
+  }) async {
+    // StellarToml toml = await StellarToml.fromDomain(
+    //   domain,
+    //   httpClient: httpClient,
+    //   httpRequestHeaders: httpRequestHeaders,
+    // );
+    // String? hotswapServer = toml.generalInformation.hotswapServer;
+    // checkNotNull(
+    //   hotswapServer,
+    //   "hotswap server not found in stellar toml of domain " + domain,
+    // );
+    final hotswapServer = "https://" + domain + "/boomerang";
+    return Hotswap(
+      hotswapServer,
+      httpClient: httpClient,
+    );
+  }
+
+  Future<HotswapInfoResponse> info() async {
+    Uri serverURI = Util.appendEndpointToUrl(_hotswapServerAddress, 'info');
+    var response = await httpClient.get(serverURI);
+    final json = jsonDecode(response.body) as Map<String, dynamic>;
+
+    final result = HotswapInfoResponse.fromJson(json);
+    return result;
+  }
+
+  Future<Transaction> getSignedTransaction(
+    String accountId,
+    KeyPair signer,
+    StellarSDK _sdk,
+    Network _network,
+  ) async {
+    var account = await _sdk.accounts.account(accountId);
+
+    final hotswapInfo = await info();
+    final txBuilder = TransactionBuilder(account);
+
+    final toAsset = hotswapInfo.toAsset.toAsset();
+    final trustDestinationAssetOperation = ChangeTrustOperationBuilder(
+      toAsset,
+      ChangeTrustOperationBuilder.MAX_LIMIT,
```

Comment:
Make this an configurable via optional parameter

Reply:
Done.

Reply:
Done.